golang.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import "fmt" //here is a line coment
+
+// FileGetter is usd to retrieve de code.
+type FileGetter interface {
+	// GetFiles returs a FilesScanner
+	GetFiles(context.Context, *FilesRequest) (FileScanner, error)
+}
+
+func main() {
+    /*
```

Comment:
Typo inside comment in token 'Multi'. Possible corrections:multi (74),mult (0),rmulti (0)